deployer/template/aws_elasticloadbalancingv2_targetgroup.go

```diff
@@ -0,0 +1,35 @@
+package template
+
+import (
+	"github.com/awslabs/goformation/cloudformation"
+	"github.com/awslabs/goformation/cloudformation/resources"
+)
+
+func ValidateAWSElasticLoadBalancingV2TargetGroup(
+	projectName, configName, resourceName string,
+	template *cloudformation.Template,
+	res *resources.AWSElasticLoadBalancingV2TargetGroup,
+) error {
+	res.Name = normalizeName("fenrir", projectName, configName, resourceName, 32)
+
+	res.Tags = append(res.Tags, resources.Tag{Key: "ProjectName", Value: projectName})
+	res.Tags = append(res.Tags, resources.Tag{Key: "ConfigName", Value: configName})
+	res.Tags = append(res.Tags, resources.Tag{Key: "ServiceName", Value: resourceName})
+
+	// Only allow lambda targets for now
+	if res.TargetType != "lambda" {
+		return resourceError(res, resourceName, "TargetGroup.TargetType must be lambda")
+	}
+
+	// Only allow lambdas created in this template.
```

Comment:
I think for this, we should support a similar lookup method as used by the sg and subnet's. I.e. allowing tags _or_ ID here.

Reply:
this is actually done, forgot to get rid of the comment

You can specify anything that `lambda.FindFunction` supports, which is either full arn, lambda name or partial arn. It verifies the tags on the lambda allow fenrir